themes/bootstrap3/templates/_ui/components/account-offcanvas-show.phtml

```diff
@@ -0,0 +1,10 @@
+<?php
+    if (null === ($side = $this->config()->offcanvasSide())) {
+        return '';
```

Comment:
I thought the intention of the show button was to serve as a jump link to the anchor in responsive mode, and an offcanvas toggle in offcanvas mode... so I think it serves a purpose in both situations. Do we really need/want to hide it?

Reply:
You're right, and I totally missed the dual purpose of it. Should be fixed now with more obvious code that'll help in the future. I'm not 100% happy with the fact that the button is defined in offcanvas.less also for non-offcanvas mode, but changing that would cause back-compatibility issues, so I suppose it's better to leave it like it is for now. I just added a comment into the less/scss files about the non-offcanvas mode.